pkg/linter/linter.go

```diff
@@ -109,10 +109,15 @@ var postLinterMap = map[string]postLinter{
 		FailOnError: false,
 		Explain:     "Verify that this package is supposed to be empty; if it is, disable this linter; otherwise check the build",
 	},
+	"pythondocs": postLinter{
+		LinterFunc:  pythonDocsPostLinter,
+		FailOnError: false,
+		Explain:     "Remove all docs directories from the package",
```

Comment:
Is there a pipeline we should provide to split them out? `python/docs` or `split/pythondocs` or something? We have `split/manpages` today, for example.

Reply:
Hmm. We could do it! I think we should do that in a separate request but I'm absolutely for it 100%. I think we should give some thought to that change.